doctrine.rst

```diff
@@ -746,6 +746,23 @@ In the example above, the ``$product`` argument is handled automatically,
 but ``$comment`` is configured with the attribute since they cannot both follow
 the default convention.
 
+If you need to get other information from the request to fetch your entity, you
```

Comment:
nitpick, but then this should be your entities, or at least your data ^^

Reply:
Reworded to match any case :ok_hand: